client/src/components/CreatePost/Form/FirstSection.js

```diff
@@ -25,12 +36,15 @@ const First = ({
         onChange={onChangeTitle}
         value={formData.title}
         placeholder="Title"
+        maxLength={35}
+        required
```

Comment:
Wait, so what's the purpose Line 40 here then?

Reply:
I initially added the red text validation, but I removed it after we talked about in the thread, however,  I forgot to remove "required" on both lines 47 and 40. The question is now Should we proceed with a red text or this minor change is ok? what are your thoughts?

Reply:
I am not sure if we need it though, I guess we could change the logic of the "Post" button? maybe? I am not sure but I think this line will only work if you tried to submit title/body empty

Reply:
Thanks for removing the visibility changes @omarzoghayyer . 

The `required` I think we should remove here and address the "required" indicator handling 
until we have a design (let's ask someone in UX/UI).  

Also I I see in the slack convo that the body input color should also be black so we should add that.

We can merge these tweaks and start a new feature branch once we have the design, but I would also unlink #1002 as that won't be resolved until we have that.

Reply:
I agree with @joshmorel. Let’s see if they can design this feature and re attempt it as whole. We might want to close this PR in this case. 

I’d like to work on it, once they come up with a design.

Reply:
Yup @joshmorel I agree. @omarzoghayyer just remove the `required` from line 40 and 47, then change the color of the body to be black as well, and then we can merge this one.

Reply:
Sounds good.